src/drivers/imx/sdma.c

```diff
@@ -442,15 +442,34 @@ static struct dma_chan_data *sdma_channel_get(struct dma *dma,
 	return NULL;
 }
 
+static void sdma_enable_event(struct dma_chan_data *channel)
```

Comment:
@paulstelian97 can you please split this patch into one patch per functional change.
One patch per fix, then I will merge them.

It will be then easier to port the patches to master.

Reply:
Done. Also if you didn't see the mail, this seems to fix both of our bugs.